### DISCUSSION TITLE
[@types/chrome] chrome.pageCapture.saveAsMHTML - callback has wrong type

### DISCUSSION THREAD

**Discussion by andrewpmontgomery** [DefinitelyTyped/DefinitelyTyped#58561] [Issues with a @types package]
Type definition doesn't match real-world code for `chrome.pageCapture.saveAsMHTML`

In PR #57867, the callback type was changed from `any` to `ArrayBuffer`.
However in practice the API actually returns `Blob`.

Here is the official documentation: https://developer.chrome.com/docs/extensions/reference/pageCapture/#method-saveAsMHTML
It says ArrayBuffer twice, but Blob once:
> The callback parameter looks like: `(mhtmlData?: ArrayBuffer) => void`
> mhtmlData
> ArrayBuffer optional
> The MHTML data as a Blob.

Easy one-line fix goes here:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7daaae06ed92bddd6f2ea62ee426431b951cfcec/types/chrome/index.d.ts#L5823

I'll raise a PR for this soon.

[a comment by an automated account is not shown]

**Comment by nicolas377** [comment]
When you open the PR, just link it here!

**Reply by andrewpmontgomery** [reply]
Here you go! https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58568
